nslsii/tests/temperature_controllers_test.py

```diff
@@ -0,0 +1,50 @@
+from nslsii.temperature_controllers import Eurotherm
+from bluesky.plan_stubs import mv
+from bluesky import RunEngine
+import subprocess
+import os
+import sys
+import pytest
+
+
+@pytest.fixture
+def RE():
+    return RunEngine()
+
+
+def test_Eurotherm(RE):
+    '''Tests the Eurotherm ophyd device.
+
+    Parameters
+    ----------
+    RE : object
+        Bluesky RunEngine for use in testing.
+    '''
+
+    stdout = subprocess.PIPE
+    stdin = None
+
+    # Start up an IOC based on the thermo_sim device in caproto.ioc_examples
+    ioc_process = subprocess.Popen([sys.executable, '-m',
+                                    'caproto.tests.example_runner',
+                                    'caproto.ioc_examples.thermo_sim'] +
+                                   list([]), stdout=stdout, stdin=stdin,
```

Comment:
Why do you need to sum an empty list here?

Reply:
you are correct this is not needed and removed.